packages/fuels-test-helpers/src/lib.rs

```diff
@@ -162,6 +163,24 @@ pub async fn setup_test_client(
     (client, srv.bound_address)
 }
 
+pub fn create_storage_slot<T: Sized, D: Sized>(key: T, value: D) -> StorageSlot {
```

Comment:
why not use `StorageSlot::new()`?

Reply:
`StorageSlot::new` accepts `Bytes32` while this would have worked with generic types. It has been removed however, see above discussion.